pkg/ottl/ottlfuncs/README.md

```diff
@@ -8,6 +8,18 @@ This document contains documentation for both types of OTTL functions:
 - [Functions](#functions) that transform telemetry.
 - [Converters](#converters) that provide utilities for transforming telemetry.
 
+## Design principles
+
+For the standard OTTL functions described in this document, we specify design principles to ensure they are always
+secure and safe for use:
+
+- Built-in OTTL functions may not access the file system, network, or any other I/O devices.
+- Built-in OTTL functions may share information only through their parameters and results.
+- Built-in OTTL functions must be terminating; they must not loop forever.
+
+OTTL functions are implemented in Go, and so are only limited by what can be implemented by a Go program.
```

Comment:
```suggestion
OTTL functions are implemented in Go, and so are only limited by what can be implemented in a Go program.
```
Just to avoid using "by" twice.